jerry-core/ecma/builtin-objects/ecma-builtin-bigint-prototype.c

```diff
@@ -125,6 +137,48 @@ ecma_builtin_bigint_prototype_object_to_string (ecma_value_t this_arg, /**< this
   return ecma_make_string_value (string_p);
 } /* ecma_builtin_bigint_prototype_object_to_string */
 
+/**
+ * Dispatcher of the built-in's routines
+ *
+ * @return ecma value
+ *         Returned value must be freed with ecma_free_value.
+ */
+ecma_value_t
+ecma_builtin_bigint_prototype_dispatch_routine (uint8_t builtin_routine_id, /**< built-in wide routine
+                                                                              *   identifier */
+                                                ecma_value_t this_arg, /**< 'this' argument value */
+                                                const ecma_value_t arguments_list_p[], /**< list of arguments
+                                                                                      *   passed to routine */
+                                                uint32_t arguments_number) /**< length of arguments' list */
+{
+  ecma_value_t this_value = ecma_builtin_bigint_prototype_object_value_of (this_arg);
```

Comment:
The `this_value` is not freed in this method. Shouldn't we release it?

Reply:
`this_value` should be released in case of `ECMA_BIGINT_PROTOTYPE_TO_STRING` and `ECMA_BIGINT_PROTOTYPE_TO_LOCALE_STRING` before breaking in the switch